client/packages/core/src/Reactor.js

```diff
@@ -360,11 +362,15 @@ export default class Reactor {
         break;
       case "join-room-ok":
         const loadingRoomId = msg["room-id"];
-        const loadingRoom = this._presence[loadingRoomId];
-        if (loadingRoom) {
-          loadingRoom.isLoading = false;
+        const joinedRoom = this._rooms[loadingRoomId];
+
+        if (!joinedRoom) {
+          break;
         }
+
+        joinedRoom.isLoading = false;
```

Comment:
Makes me also realize: what happens if we go offline and back online? 

Now each 'joinedRoom` should be set back to 'isLoading', as we will be sending new join events.